release/models/rib/openconfig-rib-bgp-tables.yang

```diff
@@ -9,6 +9,10 @@ submodule openconfig-rib-bgp-tables {
   import openconfig-inet-types { prefix oc-inet; }
   import openconfig-extensions { prefix oc-ext; }
   import openconfig-policy-types { prefix oc-pol-types; }
+  import ietf-yang-types { prefix yang; }
```

Comment:
Is this really necessary?  
There is a Openconfig-yang-types. It is not common to see IETF and Openconfig Yang mixture.

Reply:
Let me look for the same type and use it from OC-yang-types.

Reply:
@sbarguil I did some more investigation and several published models are already using `ietf-yang-types`.

For example:

* https://github.com/openconfig/public/blob/master/release/models/network-instance/openconfig-network-instance-l2.yang#L10
* https://github.com/openconfig/public/blob/master/release/models/network-instance/openconfig-network-instance-l2.yang#L261

I can change it but wanted to stay consistent with the previous work we did.